wrappers/obj-c/ODWLogConfiguration.mm

```diff
@@ -431,6 +431,11 @@ -(void)set:(nonnull NSString *)key withValue:(nonnull NSString *)value
     (*_wrappedConfiguration)[[key UTF8String]] = [value UTF8String];
 }
 
+-(void)setHost:(nonnull NSString *)host
```

Comment:
You might want to add a get method, to keep consistency with other properties functionality.